lib/linguist/heuristics.yml

```diff
@@ -699,6 +699,10 @@ disambiguations:
     pattern: '^\s*<\?xml'
 - extensions: ['.s']
   rules:
+  - language: Assembly
+    and:
+      - negative_pattern: 'mov\s+%'
+      - pattern: 'mov\s+[^%]{2,},'
```

Comment:
This regex is vulnerable to ReDoS attack. Please find a better regex which isn't that also runs linearly.

Reply:
fixed